sklearn/model_selection/_search.py

```diff
@@ -337,10 +340,12 @@ def _check_param_grid(param_grid):
                                  "list.")
 
 
+# XXX Remove in 0.20
 class _CVScoreTuple (namedtuple('_CVScoreTuple',
                                 ('parameters',
                                  'mean_validation_score',
-                                 'cv_validation_scores'))):
+                                 'cv_validation_scores',
+                                 'corrected_std'))):
```

Comment:
No, you can't add a field to a namedtuple in the public API. Surely there is current code that tries to unpack the namedtuple with three slots `for p, m, s in gs.grid_scores_`. You will break such code